kedro/contrib/io/pyspark/spark_jdbc.py

```diff
@@ -144,6 +144,15 @@ def __init__(
 
         # Update properties in load_args and save_args with credentials.
         if credentials is not None:
+
+            # Check credentials for bad inputs
```

Comment:
```suggestion
            # Check credentials for bad inputs.
```
Similar trivial feedback.